### DISCUSSION TITLE
Can't activate/dectivate copilot

### DISCUSSION THREAD

**Discussion by bonmarip** [community/community#27723] [Copilot Conversations]
The copilot button on the VSCode task bar has stopped working for me. It used to deploy a menu to ask if I wanted to deactivate/activate the github copilot completely or just for a specific language but now nothing appears and it doesn't activate/deactivate. I tried reinstalling copilot and vscode, using default settings for VSCode without success. I can still disable the copilot through its extension page but that requires a reload. I have also tried activating the copilot by setting and pressing a keyboard shorcut but without success either.
I was really enjoying learning C++ and I wish to be able to go back to it as soon as possible.
Does anyone has an idea of what's happening to me ?

**Comment by MrSarno** [comment]
That's strange.

Have you tried specifically deleting the settings.json file?

I'm not sure whether it is removed as part of the uninstallation process.

You haven't clarified the OS you're using, but it should be available in each of the following places;

Windows: `%APPDATA%\Code\User\settings.json`
macOS: `HOME/Library/Application Support/Code/User/settings.json`
Linux: `HOME/.config/Code/User/settings.json`

**Comment by bonmarip** [comment]
Thanks a lot for your response,
I've tried deleting the file as you suggested but it didn't solve my problem. For now my solution is to just change "github.copilot.inlineSuggest.enable" in settings.json whenever I need to
I'm on Windows btw
